strongbox-web-core/src/main/java/org/carlspring/strongbox/rest/SearchRestlet.java

```diff
@@ -21,9 +15,18 @@
 import javax.ws.rs.core.Response;
 import java.io.IOException;
 
+import io.swagger.annotations.*;
+import org.apache.lucene.queryparser.classic.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Component;
+
 @Component
 @Path("/search")
 @Api(value = "/search")
+@PreAuthorize("permitAll")
```

Comment:
@AlexOreshkevich ,
I think this needs to be furter fine-grained. Not everyone will be allowed to search.